MISRA.md

```diff
@@ -2,25 +2,25 @@
 
 The coreJSON library files conform to the [MISRA C:2012](https://www.misra.org.uk)
 guidelines, with some noted exceptions. Compliance is checked with Coverity static analysis.
-Deviations from the MISRA standard are listed below:
+The specific deviations, suppressed inline, are listed below.
 
-### Ignored by [Coverity Configuration](tools/coverity/misra.config)
-| Deviation | Category | Justification |
-| :-: | :-: | :-: |
-| Directive 4.9 | Advisory | Allow inclusion of function like macros. |
-| Rule 3.1 | Required | Allow nested comments. C++ style `//` comments are used in example code within Doxygen documentation blocks. |
-| Rule 8.13 | Advisory | Allow one function to have a char * argument without const qualifier. |
-| Rule 15.4 | Advisory | Allow more then one `break` statement to terminate a loop. |
-| Rule 19.2 | Advisory | Allow a `union` of a signed and unsigned type of identical sizes. |
-| Rule 20.12 | Required | Allow use of `assert()`, which uses a parameter in both expanded and raw forms. |
-
-### Flagged by Coverity
-| Deviation | Category | Justification |
-| :-: | :-: | :-: |
-| Rule 2.5 | Advisory | A macro is not used by the library; however, it exists to be used by an application. |
-| Rule 8.7 | Advisory | API functions are not used by the library; however, they must be externally visible in order to be used by an application. |
+Additionally, [MISRA configuration file](https://github.com/FreeRTOS/coreJSON/blob/main/tools/coverity/misra.config) contains the project wide deviations.
 
 ### Suppressed with Coverity Comments
-| Deviation | Category | Justification |
-| :-: | :-: | :-: |
-| Rule 11.3 | Required | False positive - the rule permits type qualifiers to be added. |
+To find the violation references in the source files run grep on the source code
+with ( Assuming rule 11.4 violation; with justification in point 2 ):
+```
+grep 'MISRA Ref 11.4.2' . -rI
+
+#### Rule 11.3
+_Ref 11.3.1_
+
+- MISRA C-2012 Rule 11.3 prohibits casting a pointer to a different type.
+        This instance is a false positive, as the rule permits the
+        addition of a type qualifier.
```

Comment:
Since this describes the specific instance of violation, could help adding the details like: Rule permits adding a `const` qualifier to cast.

Reply:
Very good point, added that change